presto-product-tests/README.md

```diff
@@ -28,3 +37,8 @@ Product tests are not run by default. To start them use run following command:
 ```
 java -jar target/presto-product-tests-*-executable.jar --config-local file://`pwd`/tempto-configuration-local.yaml
 ```
+or (more commonly)
```

Comment:
We should also add quarantine group to this command? And instead of more commonly say, or to run tests with mysql, postgres and qurantine group excluded